modules/ossm-threescale-integration-settings.adoc

```diff
@@ -15,10 +15,10 @@ For 3scale SaaS customers, {ProductName} is enabled as part of the Early Access
 
 .Procedure
 
-. Navigate to *[your_API_name]* -> *Integration* -> *Configuration*.
+. Navigate to *[your_API_name]* -> *Integration*.
 
-. At the top of the *Integration* page click on *edit integration settings* in the top right corner.
+. Click *Settings*.
 
-. Under the *Service Mesh* heading, click the *Istio* option.
+. Select the *Istio* option under _Deployment_.
 
-. Scroll to the bottom of the page and click *Update Service*.
+. Click *Update Product* to save your selection.
```

Comment:
I believe there is an additional step required:

- Go into *Configuration*
- Click *Update configuration*